### ISSUE TITLE
Add support for local storage

### ISSUE THREAD

**Issue by steven-basart** [centerforaisafety/cerberus-cluster#68]
Probably want some of the space if used to be cleared each run or at least be over written.

**Comment by steven-basart** [comment]
Slurm has support for this already:
https://slurm.schedmd.com/job_container.conf.html

Involves 
Add to slurm.conf the following:

JobContainerType=job_container/tmpfs

We already have the other line in our slurm.conf.

Within in each compute node
Create the following directory and set a quota on it.
/mnt/localdisk/slurm_tmp
We'll create slurm_tmp in all compute nodes and make it rwx by root and rwx by slurm and no permissions for anyone else.

Then creating a new file called 
/etc/slurm/job_container.conf

Inside it we'll have the following:
``` 
AutoBasePath=true
BasePath=/mnt/localdisk/slurm_tmp
```

Then we'll direct it to create the space within the local storage for each compute node.

This will give the users access to temporary storage called /tmp for them which is mounted to /dev/shv

**Comment by steven-basart** [comment]
Merged in PR #83 

Remaining work is deploy to prod.  

Steps
Add the following to cais_compute ansible playbook
```
# Used for local storage
- name: Create local storage location for all compute nodes
  become: true
  file: 
    path: /mnt/localdisk/slurm_tmp
    state: directory
    owner: root
    group: slurm
    mode: '0770'
```
Run the playbook
update slurm.conf to include
```
JobContainerType=job_container/tmpfs
```
Add job_container.conf
```
AutoBasePath=true
BasePath=/mnt/localdisk/slurm_tmp
```
Then run
```
sudo scontrol reconfigure
```

**Comment by steven-basart** [comment]
Deployed to prod marking as complete.
